File: src/cs/Demos/SolidEdge/OpenSave/SearchResultItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SolidEdge.OpenSave
{
    public struct SearchResultItem
    {
        public SearchResultItem(string file, Version version)
        {
            FileName = file;
            Version = version;
        }

        public string FileName;
        public Version Version;
    }
}

File: General/OpenSave/vb/OpenSave/SearchResultItem.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Public Structure SearchResultItem
    Public Sub New(ByVal file As String, ByVal version As Version)
        FileName = file
        Me.Version = version
    End Sub

    Public FileName As String
    Public Version As Version
End Structure
